swift-generator-cli/src/main/java/com/facebook/swift/generator/SwiftGeneratorCommandLineConfig.java

```diff
@@ -86,4 +86,10 @@
             description = "Include paths are relative to the file in which they are contained"
     )
     public boolean includePathsRelativeToFile = false;
+
+    @Parameter(
+            names = "-package_prefix",
```

Comment:
Actually I wanted to that originally, but then I realized that it can be confused to mean only prefix the `java` namespace and not `java.swift` namespace, since there is another flag named `use_java_namespace` 😕